mkosi/config.py

```diff
@@ -595,6 +596,24 @@ def __call__(
         parser.exit()
 
 
+class PagerDocAction(argparse._HelpAction):
+    def __call__(
+        self,
+        parser: argparse.ArgumentParser,
+        namespace: argparse.Namespace,
+        values: Union[str, Sequence[Any], None] = None,
+        option_string: Optional[str] = None
+    ) -> None:
+        with importlib.resources.path("mkosi.resources", "mkosi.1") as man:
+            if man.exists():
+                run(["man", man])
+                parser.exit()
+
+        md = importlib.resources.read_text("mkosi.resources", "mkosi.md")
+        page(md, namespace.pager)
+        parser.exit()
```

Comment:
I would change this to only show the man page and error out if the man page is not available. Or even better, generate it on demand from mkosi.md if pandoc is installed and error out if pandoc is not installed. But I don't think we should show the raw markdown file here tbh.

Reply:
I thought about falling back to pandoc if no man page is available and had the code in there in one version, but I'm not sure we should hardcode that here, since it would make pandoc a recommends or similar in most distros.

Reply:
That should be fine. If the distro puts the man page in the appropriate place, then the pandoc dependency is not needed.